dask/array/ma.py

```diff
@@ -170,3 +172,21 @@ def set_fill_value(a, fill_value):
 @derived_from(np.ma)
 def average(a, axis=None, weights=None, returned=False):
     return _average(a, axis, weights, returned, is_masked=True)
+
+
+def _chunk_count(x, axis=None, keepdims=None):
+    return np.ma.count(x, axis=axis, keepdims=keepdims)
+
+
+@derived_from(np.ma)
+def count(a, axis=None, keepdims=False, split_every=None):
+    return reduction(
+        a,
+        _chunk_count,
+        chunk.sum,
+        axis=axis,
+        keepdims=keepdims,
+        dtype=np.intp,
```

Comment:
@jcrist I think this is the route of the windows dtype issue. There is something slightly different that you'll want to do. Looking now...

Reply:
`np.ma.count` passes in `dtype=np.intp` directly, which is why I made this change in the first place:

https://github.com/numpy/numpy/blob/56d9194bef1a76f86cbb376cbf4954ca653f9d45/numpy/ma/core.py#L4583

What's weird is that it's dask returning an `int32` and numpy returning an `int64` - usually the windows dtype errors like this go the other way.

Anyway, if you have an idea of what to change I'd love to hear it - not looking forward to booting up my windows box.

Reply:
Eh nevermind, you could try wrapping this in a `np.dtype()` but I think what you have should already be right.

Reply:
@jcrist - since neither of us got to it last week should we just skip the test that is failing and merge this?

Reply:
I just pushed a commit (e0039dc1ec143665ab8805414cae26c2800c9d73) that hopefully will get this passing.

Reply:
Thanks @jsignell and @jcrist  - much appreciated.